client-side/src/Components/TableComponent.jsx

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react';
+import { DataGrid } from '@mui/x-data-grid';
+import PropTypes from 'prop-types'
+import './TableComponent.scss';
+
+const TableComponent = ({ dataObject, widthOfTable = "80%" }) => {
+  const columns = dataObject.headers.map((header) => ({
+    field: header,
+    headerName: header,
+    width: 150,
+  }));
+
+  return (
+    <div className="table" style={{ width: widthOfTable }}>
```

Comment:
please move styles to scss file

Reply:
widthOfTable is a variable that come from storybook so i cant move it to scss page

Reply:
you can declare the default 80% as variable.
also the 150 const should be a variable

Reply:
i dont understand why?
and where declare this variable?